pages/pipelines/controlling_concurrency.md

```diff
@@ -43,6 +43,9 @@ For example, if you have two pipelines that each deploy to a different target bu
 
 Concurrency groups guarantee that jobs will be run in the order that they were created in. Jobs inherit the creation time of their parent. Parents of jobs can be either a build or a pipeline upload job. As pipeline uploads add more jobs to the build after it has started, the jobs that they add will inherit the creation time of the pipeline upload rather than the build.
 
+>🚧 <code>concurrency_group</code> and <code>block</code>/<code>input</code> steps
+> Note that both [block](/docs/pipelines/block-step) and [input](/docs/pipelines/input-step) steps cause the steps to be uploaded and scheduled at the same time, which breaks concurrency groups: it blocks jobs being added to the concurrency group, but does not adjust ordering once unblocked. The concurrency won't be added to the queue until the block step is unblocked, and once it does, the timestamp will be from the pipeline upload step.
```

Comment:
Hi @pzeballos , since this subject matter is still a tad new to me, I just a few queries I'd like to make to help clarify this note...

When you say:
> cause the _steps_ to be uploaded and scheduled at the same time

Do these _steps_ refer to only 'block' and 'input' steps and if so, is this only 'block' and 'input' steps in a single pipeline or those defined across many pipelines (which are defined as part of the same concurrency group)? Alternatively, do these _steps_ refer to any type of step (in a single pipeline or many pipelines), which are defined as part of the same concurrency group?

Reply:
Sorry @gilesgas! We are still having customers raising this issue os having the note in the docs should help a but :) 

The issue is with both, block and input steps. Concurrency group groups work across pipelines; what it matters is the name of the queue (ie, having the same concurrency attribute)

Reply:
Thanks @pzeballos , I can now revisit this PR and should be able to have it merged in soon.

To speed things along, I'll push some updates directly to this PR myself...